pkg/apis/gceproviderconfig/v1alpha1/gcemachineproviderconfig_types.go

```diff
@@ -32,11 +32,13 @@ type GCEMachineProviderConfig struct {
 	Roles []MachineRole `json:"roles,omitempty"`
 
 	Zone        string `json:"zone"`
-	MachineType string `json:"machineType"`
+	MachineType string `json:"machineType,omitempty"`
```

Comment:
Can we also add project here?

Reply:
Project is part of cluster config. I don't think we need to duplicate this here. Or do you have another use case in mind?